code/modules/research/techweb/all_nodes.dm

```diff
@@ -150,8 +150,17 @@
 	description = "Deeper understanding of how the Bluespace dimension works"
 	prereq_ids = list("practical_bluespace", "high_efficiency")
 	design_ids = list("bluespace_matter_bin", "femto_mani", "triphasic_scanning", "tele_station", "tele_hub", "quantumpad", "launchpad", "launchpad_console",
-	"teleconsole", "bag_holding", "bluespace_crystal", "wormholeprojector", "bluespace_pod")
-	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 15000)
+	"teleconsole", "bluespace_crystal", "bluespace_pod", "wormholeprojector")

```

Comment:
wormholeprojector is grief, don't make it cheaper reee

Reply:
fix the gun instead lol

Reply:
That's the whole point of the gun, there's nothing to fix

Reply:
How were they nerfed? I'm the one who raised them to 15K because of their compound nature and very little to do with BoHs

Reply:
Put this at 12,500 and the BoH at 2,500 as a node after this if you insist on touching the node.

Reply:
10-5 is fine with me as well, if it becomes a problem I'll shift it back.

Reply:
this was high due to t4 stock parts.

Reply:
It was